### PR TITLE
Part 1 of issue 1555: Update layout to use numbered sections

### PR DESCRIPTION
**Pull request by jernestmyers**
Addresses #1556 as part 1 of 3 outlined in #1555 

image

### CODE REVIEW

File: src/lib/core/components/computations/ComputationStepContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useUITheme } from '@veupathdb/coreui/dist/components/theming';
+import {
+  NumberedHeader,
+  NumberedHeaderProps,
+} from '../../../workspace/Subsetting/SubsetDownloadModal';
```
Comment (asizemore) [inline src/lib/core/components/computations/ComputationStepContainer.tsx:6]: maybe we should move `NumberedHeader` to CoreUI someday?

File: src/lib/core/components/computations/RunComputeButton.tsx [hunk 2]
```diff
@@ -20,11 +19,14 @@ export function RunComputeButton(props: Props) {
...
         alignItems: 'center',
+        padding: '1em 0',
+        marginLeft: '3em',
+        marginBottom: '2em',
       }}
     >
       <FilledButton
         themeRole="primary"
-        text="Run computation"
+        text={`Generate ${computationAppOverview.displayName} data`}
```
Comment (asizemore) [inline src/lib/core/components/computations/RunComputeButton.tsx:29]:
```suggestion
        text={`Generate ${computationAppOverview.displayName} results`}
```
There was some recent discussion that "results" would be more clear to the user than "data" in all apps.

File: src/lib/core/components/computations/plugins/abundance.tsx [hunk 3]
```diff
@@ -171,45 +179,40 @@ export function AbundanceConfiguration(props: ComputationConfigProps) {
...
-    <div
-      style={{
-        display: 'flex',
-        gap: '0 2em',
-        padding: '1em 0',
-        alignItems: 'center',
+    <ComputationStepContainer
+      computationStepInfo={{
+        stepNumber: 1,
+        stepTitle: `Configure ${computationAppOverview.displayName} data`,
```
Comment (asizemore) [inline src/lib/core/components/computations/plugins/abundance.tsx:185]:
```suggestion
        stepTitle: `Configure ${computationAppOverview.displayName}`,
```

Another recent minor wording change for all apps :)

File: src/lib/core/components/visualizations/VisualizationsContainer.tsx [hunk 4]
```diff
@@ -686,50 +686,87 @@ export function FullScreenVisualization(props: FullScreenVisualizationProps) {
...
-                {/* Add image for some compute statuses} */}
-                {computeJobStatus == 'no-such-job' && <EmptyPlotSVG />}
-                {!isTerminalStatus(computeJobStatus) && (
-                  // <img style={{width:600}} src={relaxMicrobe}/>
-                  <RelaxMicrobeSVG />
+          {computationAppOverview.computeName ? (
+            <ComputationStepContainer
+              computationStepInfo={{
+                stepNumber: 3,
+                stepTitle: `Use ${computationAppOverview.displayName} data in visualization`,
```
Comment (asizemore) [inline src/lib/core/components/visualizations/VisualizationsContainer.tsx:709]:
```suggestion
                stepTitle: `Use ${computationAppOverview.displayName} results in visualization`,
```

**Review by asizemore — APPROVED** [review]
Looks great! Works great!

File: src/lib/core/components/computations/ComputationStepContainer.tsx (hunk 1 above)
Comment (jernestmyers) [reply src/lib/core/components/computations/ComputationStepContainer.tsx:6]: I created an issue for this https://github.com/VEuPathDB/CoreUI/issues/137

**Comment by jernestmyers** [comment]
Should wait until after monorepo migration.

**Pull request merged**